examples/graphene_apps/cppopenvino/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.5)
+set(OPENVINOWL openvinowl)
+project(${OPENVINOWL} CXX)
+
+# Use this to globally use C++11 with in our project
+set(CMAKE_CXX_STANDARD 11)
+
+# Load in pkg-config support
+find_package(PkgConfig)
+# Use pkg-config to get hints for 0mq locations
+pkg_check_modules(PC_ZeroMQ QUIET zmq)
+
+# find where 'zmq.hpp' is located
```

Comment:
We can do it 2 ways. 
1)Either by installing zmq in standard path. Then I guess above CMAKE directives for ZMQ are not required.  
2) I came across a link which suggested to use CMAKE directive to find zmq package since that's more generic and will work irrespective of where ZMQ is installed. 

I don't have a preference on approach 1 or 2. But let me know if you prefer 1 or 2. I can make changes accordingly.

Reply:
2 is generic but not efficient. If we were installing the zmq cpp package in standard path then choose 1 which is efficient.

Reply:
I can make changes for 1. But what is "not efficient" means ? It's a build time binding and not and not time.